common/spec/dependabot/pull_request_creator/message_builder/link_and_mention_sanitizer_spec.rb

```diff
@@ -83,6 +83,15 @@
         it { is_expected.to eq("<p>Great work <code>@greysteil</code>!</p>\n") }
       end
 
+      context "that appears in parens" do
+        let(:text) { "by Esben Sparre Andreasen (@esbena)" }
+        it do
+          is_expected.to eq(
+            "<p>by Esben Sparre Andreasen (<code>@esbena</code>)</p>\n"
```

Comment:
I think the current sanitizer will turn this into a link like `(<a href="https://github.com/esbena">@esbena</a>)`?

Reply:
Correct, it seems like people are getting mentions from that. I'm going to test that a bit further and if it's the case, get rid of the linking entirely

Reply:
Hmm, I don' t seem to get mentioned when I try this on a test account, I'm confused as to what's happening, but this change would actually make it worse as it won't wrap in codeblocks